### ISSUE TITLE
Why tax terms & site options vs a custom post type?

### ISSUE THREAD

**Issue by jtsternberg** [hlashbrooke/Code-Snippet-Library#1]
Saving the snippets as a taxonomy term and using site options to save the associated data seems like a misuse of taxonomies and site options. These snippets are a perfect candidate for a custom post type. Any chance you'll update or accept a pull request?

**Comment by hlashbrooke** [comment]
I agree that these snippets would more logically be saved as a CPT, but the reason I chose to do it as a taxonomy is that the taxonomy UI is far better and more efficient for this kind of thing. The CPT UI does not make sense for small items like this as it is slow and cumbersome. If you have a compelling reason why I should change it then I'm all ears :)
